python/ray/rllib/agents/impala/vtrace_policy_graph.py

```diff
@@ -140,30 +142,28 @@ def __init__(self,
 
         if isinstance(action_space, gym.spaces.Discrete):
             is_multidiscrete = False
-            actions_shape = [None]
             output_hidden_shape = [action_space.n]
         elif isinstance(action_space, gym.spaces.multi_discrete.MultiDiscrete):
             is_multidiscrete = True
-            actions_shape = [None, len(action_space.nvec)]
             output_hidden_shape = action_space.nvec.astype(np.int32)
         else:
-            raise UnsupportedSpaceException(
-                "Action space {} is not supported for IMPALA.".format(
-                    action_space))
+            is_multidiscrete = False
+            output_hidden_shape = 1
```

Comment:
why does `output_hidden_shape` be 1 in this branch

Reply:
I believe this is the equivalent of `[n]` in the normal Discrete case above, which doesn't actually cause any splitting to happen below (splitting is only for multi-discrete).